webapp/src/components/channel_call_toast/component.tsx

```diff
@@ -49,7 +50,7 @@ export default class ChannelCallToast extends React.PureComponent<Props, State>
         if (this.props.connectedID) {
             return;
         }
-        window.postMessage({type: 'connectCall', channelID: this.props.currChannelID}, window.origin);
+        window.postMessage({type: 'connectCall', channelID: this.props.currChannelID, teamID: this.props.currTeamId}, window.origin);
```

Comment:
Is this redundant? I see we are loading the team regardless of whether it's passed in the event handler.

Reply:
Yes, it is, I'll remove this.